modules/operator-upgrade-modify-central-custom-resource.adoc

```diff
@@ -0,0 +1,30 @@
+// Module included in the following assemblies:
+//
+// * upgrading/upgrade-operator.adoc
+:_module-type: PROCEDURE
+[id="operator-upgrade-modify-central-custom-resource_{context}"]
+= Modifying Central custom resource
+
+[role="_abstract"]
+If you have not configured a default storage class for {product-title-short} that is SSD or high performance, you must update the `Central` custom reource to request persistent storage by creating a persistent volume claim (PVC).
+
+[NOTE]
+====
+Skip this section if you have already configured a default storgae class for {product-title-short}.
+====
+
+.Procedure
+* Update the sentral custom resource with the following configuration:
+[source,terminal]
+----
+central:
+      db:
+        isEnabled: Default <1>
+        persistence:
+          persistentVolumeClaim: <2>
+            claimName: central-db
+            size: 100Gi
+            storageClassName: <optional>
+----
+<1> You must not change the value of `IsEnabled` to `Enabled`.
```

Comment:
@msugakov The goal was to have the deployment of 4.0 succeed when the user makes the channel switch. Otherwise if the user doesn't have a default storage class, then central-db pvc would not be created. It's not *required* in the sense that the reconciliation of 4.0 may just fail then they'd have to fix the CR to specify persistence settings